sample/ingress-controller-tls/README.md

```diff
@@ -37,6 +37,7 @@ helm install stable/nginx-ingress --generate-name \
 
 - To provide identity to access key vault, refer to the following [section](https://github.com/Azure/secrets-store-csi-driver-provider-azure#provide-identity-to-access-key-vault).
 - Set the `tenantId` and `keyvaultName`
+- Use `objectType: secret` for `ingresscert` as this is the only way to retrieve the certificate and private key from azure key vault as documented [here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/docs/getting-certs-and-keys.md#getting-certificates-and-keys-using-azure-key-vault-provider)
```

Comment:
maybe also add a note about setting the secret type to `kubernetes.io/tls`

Reply:
To address #104, in this section of the readme https://github.com/Azure/secrets-store-csi-driver-provider-azure#optional-sync-with-kubernetes-secrets, we can add:
> To view an example of type `kubernetes.io/tls`, refer to the [ingress-controller-tls sample](sample/ingress-controller-tls/README.md)

Reply:
thank you, updated!